latina-in-tech/latina-in-tech.github.io
# Add feedback form

# 📚 Description
## What
- Added feedback button in main page
- created a new page for feedbacks
- linked project to a firebase firestore

Refer to screenshots for further explanations
## Why
We want to have a single endpoint for community member feedbacks
# 🧪 Testing Guide
- Clone this branch & start app in local
- Go to [firebase dashboard](https://firebase.google.com/) 
- Login with Latina In Tech Google Account
- replace `src/service/firebaseConfig.ts `variables with real firebase ones
- Open the feedback page in the app
- Fill in the fields & submit feedback
- You should be able to see the feedback entity in the firestore dashboard on firebase console

# 🖼 Screenshots/Recordings
![Screenshot Capture - 2023-12-01 - 00-25-59](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/1c9b4848-3c04-4cc5-87f7-f29548b72fdf)
![Screenshot Capture - 2023-12-01 - 00-26-27](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/13117d0b-2431-443d-9c7f-17b6008ffa01)
![Screenshot Capture - 2023-12-01 - 00-26-57](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/d1445ff0-f817-46bc-b9ca-287131015650)
![Screenshot Capture - 2023-12-01 - 00-27-31](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/7020da70-914c-4f09-84ea-708784e47eb7)
![Screenshot Capture - 2023-12-01 - 00-27-42](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/1244e3f4-f01a-4d71-9a62-4ded50e0dc58)
![Screenshot Capture - 2023-12-01 - 00-28-07](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/805dd021-5779-4397-90b0-80feefbd75dd)


# 📝 To Do
- Create a popup that will ask the user if they want to leave a feedback right after they open the main page of the app
